src/components/Navbar/index.css

```diff
@@ -122,6 +126,7 @@
 .searched_user_sub_container {
   padding: 0;
   width: 100%;
+  margin-left: 10px;
```

Comment:
I doubt we need this now as this component has been updated, can you please check?